mslib/mscolab/file_manager.py

```diff
@@ -100,10 +100,7 @@ def create_operation(self, path, description, user, last_used=None, content=None
             data = fs.open_fs(self.data_dir)
             data.makedir(operation.path)
             operation_file = data.open(fs.path.combine(operation.path, 'main.ftml'), 'w')
-            if content is not None:
-                operation_file.write(content)
-            else:
-                operation_file.write(mscolab_settings.STUB_CODE)
+            operation_file.write(content)
             operation_path = fs.path.combine(self.data_dir, operation.path)
```

Comment:
with your change you need to indent this under the if content is not None: condition